docs/ci-integration/guides/kubernetes.md

```diff
@@ -0,0 +1,117 @@
+# Kubernetes
+
+## Overview
+
+Kubernetes isn't a CI per-se, but it _can_ serve as the underpinning for many modern CI systems. As such, this example serves as a bare-bones example to base your implementations on.
```

Comment:
I would maybe clarify early on that Kubernetes support is limited to a single build node at a time and that multi-node is coming in the future. Many people looking for Earthly k8s support will want the benefits of high parallelism / builds across a cluster - which we don't have.

Reply:
Added an additional line about this in the following compatibility section.